General/overview.xml

```diff
@@ -35,4 +35,12 @@
             <page family="RX" html="https://www.renesas.com/document/mat/smart-configurator-users-manual-rx-api-reference"/>
 	    <page family="ASSP_RISCV" html="https://www.renesas.com/document/mat/smart-configurator-users-manual-risc-v-mcu-api-reference"/>
     </apimanual>
+    <faq>
+	    <page language="en_US" html="https://en-support.renesas.com/knowledgeBase#"/>
+	    <page language="ja" html="https://ja-support.renesas.com/knowledgeBase"/>
+    </faq>
+    <sc_faq>
```

Comment:
E: instead of defining new tag, can we use another attribute of <page>?
e.g. <page topic="main" language=...>
<page topic="sc" language=...>

Reply:
I updated.